src/antares/craft/model/output.py

```diff
@@ -9,9 +9,96 @@
 # SPDX-License-Identifier: MPL-2.0
 #
 # This file is part of the Antares project.
+from enum import Enum
+from typing import Any
+
+import pandas as pd
+
 from pydantic import BaseModel
 
 
+class QueryFile(Enum):
```

Comment:
You're missing the value ID = "id"
I would also prefer to rename STSTORAGE ST_STORAGE